CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ CHANGELOG
   - Although this could be consider a bug fix, it changes what columns are selected and if your code as a side-effect dependent on all columns being selected, it will break
 
 ### Added
+- New `make:graphql:input` command
 - New `make:graphql:union` command
 - New `make:graphql:interface` command
 - New `make:graphql:field` command
```

Comment:
Note: all of them were added recently but some of them very trivial so I didn't create PRs for each of them. Maybe should have 🤷‍♀️